File: src/main/java/com/webproject/service/FinancialService.java
package com.webproject.service;

import java.util.Date;
import java.util.List;

import com.webproject.entity.Financial;
import com.webproject.entity.Finandetail;
import com.webproject.myentity.FinandetailCustomer;
import com.webproject.util.PageResult;
import com.webproject.util.Result;

public interface FinancialService {
	
	public PageResult findPage(Financial financial, int pageNum, int pageSize) ;

	public Result add(Financial financial);

	public Financial findOne(String id);

	public Result update(Financial financial);

	public PageResult detailList(FinandetailCustomer search, int page, int rows);

	public Result add(Finandetail financial);

	public Result update(Finandetail financial);

	Finandetail  oneDetail(String finandetailid);

	public Result updateDetailDelete(String[] ids);

	public Result deleteDetail(String id);
	

	public List<FinandetailCustomer>  searchCharByNeay(Long userid, String dateString);

	public List<FinandetailCustomer> searchCharByNeayAndMonth(Long userid, String year, String month);

}

File: src/main/webapp/js/service/financialService.js
//服务层
app.service('financialService',function($http){
	//var AppUrl = 'http://localhost:8040/webProject';    
	var AppUrl = 'https://www.zzw777.com/webProject';
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get( '/contentCategory/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../contentCategory/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get(AppUrl +  '/financial/belogin/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post(AppUrl +  '/financial/belogin/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post(AppUrl+ '/financial/belogin/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../contentCategory/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post(AppUrl+ '/financial/belogin/searchList?page='+page+"&rows="+rows, searchEntity);
	}    	
});
